File: lab4 mpp/TeledonDomain/Donator.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace teledon.domain
{
    public class Donator : Entity<long>
    {

        public string Nume { get; set; }
        public string Prenume { get; set; }
        public string Telefon { get; set; }
        public string Adresa { get; set; }

        public override string ToString()
        {
            return Nume + " " + Prenume + ", " + Adresa.ToString() + ", " + Telefon;
        }

        public string getNumeComplet()
        {
            return this.Nume+this.Prenume;
        }

        public Donator(string Adresa, string Telefon, string Nume, string Prenume)
        {
            this.Nume = Nume;
            this.Prenume = Prenume;
            this.Adresa = Adresa;
            this.Telefon = Telefon;
        }
    }
}

File: GUImpp/TeledonModel/src/main/java/teledon/model/Donator.java
package teledon.model;

public class Donator extends Entity<Long>{

    public String nume;
    public String telefon;
    public String prenume;
    public String adresa;

    public Donator(String adresa, String telefon, String nume, String prenume){
        this.adresa = adresa;
        this.telefon = telefon;
        this.nume = nume;
        this.prenume = prenume;
    }

    public Donator() {

    }


    public void setNume(String nume) {
        this.nume = nume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public void setAdresa(String adresa) {
        this.adresa = adresa;
    }
    public void setTelefon(String telefon) {
        this.telefon = telefon;
    }
    public String getNumeComplet() {
        return getNume() + " " + getPrenume();
    }

    @Override
    public String toString() {
        return this.nume + " " + this.prenume + ", " + this.adresa + ", " + this.telefon;
    }

    public String getNume() {
        return this.nume;
    }
    public String getPrenume() {
        return this.prenume;
    }
    public String getAdresa() {
        return this.adresa;
    }
    public String getTelefon() {
        return this.telefon;
    }
}
